tests/WORKSPACE

```diff
@@ -93,7 +93,24 @@ load("@annex//:defs.bzl", annex_pinned_maven_install = "pinned_maven_install")
 
 annex_pinned_maven_install()
 
-scala_register_toolchains()
+scala_register_toolchains(
+    default_scala_toolchain_name = "test_zinc_2_13",
+    toolchains = [
+        "@@//dependencies/indirect:test_zinc_2_13_direct_deps_off",
```

Comment:
Do we need the `@@` for these or can we just use the single `@`?

Reply:
You're right—it can. Do you know the difference between `@//...` and `@@//...`? I know the difference between `@/foo//...` and `@@foo//...`, but are  `@//...` and `@@//...` the same?

Reply:
@@ is canonical name. Was introduced with bzlmod. AFAIK it's intended to be machine readable rather than human readable. https://bazel.build/rules/lib/builtins/Args.html

> [Label](https://bazel.build/rules/lib/builtins/Label.html) objects are turned into a string representation that resolves back to the same object when resolved in the context of the main repository. If possible, the string representation uses the apparent name of a repository in favor of the repository's canonical name, which makes this representation suited for use in BUILD files. While the exact form of the representation is not guaranteed, typical examples are //foo:bar, @repo//foo:bar and @@canonical_name+//foo:bar.bzl.

Reply:
Also https://bazel.build/concepts/labels

This link explains it better.